x-pack/filebeat/input/httpjson/input.go

```diff
@@ -222,6 +222,9 @@ func getRateLimit(header http.Header, rateLimit *RateLimit) (int64, error) {
 				if err != nil {
 					return 0, errors.Wrapf(err, "failed to parse rate-limit reset value")
 				}
+				if time.Unix(epoch, 0).Sub(time.Now()) <= 0 {
```

Comment:
```suggestion
				if time.Unix(epoch, 0).Before(time.Now()) {
```

Reply:
Before will not work here because we need to handle the case where the times are equal too.